<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Steganograph</class>
 <widget class="QWidget" name="Steganograph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>381</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>607</width>
    <height>381</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>607</width>
    <height>381</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Steganograph</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>381</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QPlainTextEdit" name="InputTextArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>581</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="InputTextButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>320</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Встроить</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>391</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Введите текст на латинице:</string>
     </property>
    </widget>
    <widget class="QLabel" name="CounterSymb">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="OpenFile_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>320</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Открыть</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SaveFile">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>320</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QTextBrowser" name="PrintTextArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>581</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="PrintText">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>320</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Вывод текста</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Изъятый текст:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="OpenFile">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>320</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Открыть</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Страница</string>
    </attribute>
    <widget class="QLineEdit" name="InputKeyArea">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ChangeKey">
     <property name="geometry">
      <rect>
       <x>342</x>
       <y>190</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Изменить ключ</string>
     </property>
    </widget>
    <widget class="QLabel" name="Status">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>200</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Редактирование ключа шифрования</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
